openmdao/docs/openmdao_book/features/debugging/listing_variables.ipynb

```diff
@@ -31,11 +31,13 @@
     "\n",
```

Comment:
<p>does this block need {eval-rst}?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>